modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -8,21 +8,68 @@ This guide contains an overview of the changes in Canvas Kit v9. Please
 [reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
 any questions.
 
+- [Glossary](#glossary)
+  - [Main](#main)
+  - [Preview](#preview)
+  - [Labs](#labs)
 - [Codemod](#codemod)
-- [Component Deprecations](#component-deprecations)
+- [Deprecations and Removals](#deprecations-and-removals)
   - [Drawer](#drawer)
   - [Layout and Column](#layout-and-column)
-  - [Stack](#stack-hstack-vstack)
-- [Component Updates](#component-updates)
-  - [Buttons](#buttons) - [Toast](#toast)
-- [Utility Updates](#utility-updates)
+  - [Stack, HStack, and VStack](#stack-hstack-and-vstack)
   - [focusRing](#focusring)
-  - [useTheme and getTheme](#useTheme-and-getTheme)
-  - [useCanvasTheme and getCanvasTheme](#useCanvasTheme-and-getCanvasTheme)
-  - [useThemedRing](#useThemedRing)
-  - [useThemeRTL](#useThemeRTL)
+  - [useCanvasTheme and getCanvasTheme](#usecanvastheme-and-getcanvastheme)
+  - [useThemeRTL](#usethemertl)
 - [Token Updates](#token-updates)
   - [Depth](#depth)
+- [Component Updates](#component-updates)
+  - [Button](#button)
+  - [Toast](#toast)
+- [Utility Updates](#utility-updates)
+  - [useTheme and getTheme](#usetheme-and-gettheme)
+  - [useThemedRing](#usethemedring)
+
+## Glossary
```

Comment:
thoughts on making this a [definition list mdx](https://www.markdownguide.org/extended-syntax/#definition-lists)

Reply:
Storybook's MD processor doesn't seem to support that syntax unfortunately. I did try other approaches without using a heading, but the multi-paragraph definitions for the packages seemed to work best with headings to mark each block of content. Open to other ideas!

Reply:
😞 Ok - I am interested in how this will look on the Canvas Site. To me in storybook it just blends in and I'm not really reading it any differently than the rest of the Upgrade Guide. Hence the suggestion in finding a different way to format it.